src/ngraph_rewrite_pass.cc

```diff
@@ -29,6 +29,11 @@
 #include "ngraph_utils.h"
 #include "tf_graph_writer.h"
 
+#if defined(NGRAPH_TF_ENABLE_VARIABLES_AND_OPTIMIZERS)
+#include "ngraph_enter_in_catalog.h"
+#include "ngraph_replace_variable_modifiers.h"
```

Comment:
They can be included from outside the enable_variable_ops/ ?

Reply:
Generally i was trying to move out everything I could from the `src/enable_variable_ops` folder to `src` in the general effort to merge normal + enable_variable_ops

Are you saying we merge files in normal + enable_variable_ops that already existed in normal, but for new files (like `ngraph_enter_in_catalog.h`) we keep them in `src/enable_variable_ops`. That is a good idea too, in case we want to sort of logically partition `src` so that `src` doesn't become a huge file dump?